src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -22,6 +22,7 @@ exports[`should create default config 1`] = `
     "initialAdminUser": {
       "password": "unleash4all",
       "username": "admin",
+      "email": "admin@unleash.com",
```

Comment:
We don't own unleash.com, so it would probably be better to do something like "admin@getunleash.io". Or maybe something like "admin@yourunleash.com"? Depending on what we want to change it to, that would obviously apply to all the places that use that email in this PR.

```suggestion
      "email": "admin@getunleash.io"
```

Reply:
Good catch!

Reply:
I would rather keep our official domain out of it. we can use example@unleash.run instead.